docs/how_to/debug_beaker_tests.md

```diff
@@ -1,19 +1,23 @@
-## What's Pry?
+# Debug beaker tests
 
-[Pry](http://pryrepl.org/) is a powerful Ruby editing and debugging tool.  Beaker uses Pry runtime invocation to create a developer console.
+beaker includes [pry-byebug](https://github.com/deivid-rodriguez/pry-byebug), a gem that combines two powerful related tools: pry and byebug
+
+## Pry
 
-## Set it up!
+### What's Pry?
+
+[Pry](http://pryrepl.org/) is a powerful Ruby editing and debugging tool.  Beaker uses Pry runtime invocation to create a developer console.
 
-Pry is included in the Beaker 1.13+ development gems and is available by default.  
+### Set it up!
 
-Pre-1.13.0 add Pry with `gem install pry`, then add Pry to individual tests by adding `require 'pry'` as the first line in the Ruby test file.
+Add Pry to individual tests by adding `require 'pry'` as the first line in the Ruby test file.
```

Comment:
I'm not sure if you're recommending that readers of this document edit beaker.rb, or that we could commit and release a 'require pry' statement in beaker.rb

If it's the former; that seems like bad practice for users that are running beaker from a gem rather than from source - code in a gem install folder isn't under version control and you can get yourself into weird situations